BraveRewardsUI/Settings/Auto-Contribute/Details/AutoContributeDetailsViewController.swift

```diff
@@ -382,3 +382,28 @@ extension AutoContributeDetailViewController {
     }
   }
 }
+
+/// Ledger Observers
+extension AutoContributeDetailViewController {
+  func setupLedgerObservers() {
+    ledgerObserver.excludedSitesChanged = { [weak self] key, exclude -> Void in
+      guard let self = self, self.isViewLoaded else {
+        return
+      }
+      let tableView = self.contentView.tableView
+      switch exclude {
+      case .all:
+        tableView.reloadData()
+      case .excluded:
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.3, execute: {
```

Comment:
Is this delay still needed without animations?

Reply:
Yes, the rewards db can take up some time to update the list.